dashboard/app/models/paired_user_level.rb

```diff
@@ -17,4 +17,17 @@
 class PairedUserLevel < ActiveRecord::Base
   belongs_to :navigator_user_level, class_name: 'UserLevel'
   belongs_to :driver_user_level, class_name: 'UserLevel'
+
+  # @param user_level_ids (Array) an array of user_level_ids.
+  # @return (Array) a subarray of user_level_ids containing those IDs associated
+  #   with pair programming (as driver or navigator)
+  def self.pairs(user_level_ids)
+    drivers = PairedUserLevel.
+      where(driver_user_level_id: user_level_ids).
+      pluck(:driver_user_level_id)
+    navigators = PairedUserLevel.
+      where(navigator_user_level_id: user_level_ids).
+      pluck(:navigator_user_level_id)
+    return drivers | navigators
+  end
```

Comment:
My concern with this method is that I don't think we have appropriate authorizations set up on this model. That said, I don't think this change really makes things much worse.